contracts/test/MockEzEthPool.sol

```diff
@@ -0,0 +1,247 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
In the future, it would be better for this to be a standalone PR. I get why it isn't considering this hasn't been merged, but we could base the PR containing this on top of the deployment tools.

Reply:
Yup good call, I'll pull the contracts out and land separately and then this PR can rebase to get the contracts.